rest-api/model/site.py

```diff
@@ -27,6 +27,8 @@ class Site(Base):
   city = Column('city', String(255))
   state = Column('state', String(2))
   zipCode = Column('zip_code', String(10))
+  latitude = Column('latitude', Float)
```

Comment:
I'm confused by this. latitude and longitude are already here:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/model/site.py#L21

What's going on with this?